src/psyclone/domain/lfric/transformations/lfric_loop_fuse_trans.py

```diff
@@ -158,9 +119,18 @@ def validate(self, node1, node2, options=None):
         '''
         # pylint: disable=too-many-locals,too-many-branches
         # Call the parent class validation first
+
+        if not options:
+            options = {}
+        same_space = options.get("same_space", False)
+        if same_space and not isinstance(same_space, bool):
+            raise TransformationError(
+                "Error in {0} transformation: The value of the 'same_space' "
+                "flag must be either Boolean or None type, but the type of "
```

Comment:
I think in Python it is `bool` so please s/Boolean/bool/

Reply:
Fixed (including the corresponding test).